cypress/integration/singlePetPage_spec.js

```diff
@@ -1,27 +1,55 @@
 describe('Pet Applications Component', () => {
+  beforeEach(() => {
```

Comment:
So great that this worked on the first go. We put in the hours this morning to figure it out, and it worked seamlessly! Great work.